src/cmd/gen-sops.ts

```diff
@@ -31,6 +31,11 @@ const providerMap = {
 
 export const genSops = async (): Promise<void> => {
   const argv: BasicArguments = getParsedArgs()
+  const targetPath = `${env.ENV_DIR}/.sops.yaml`
```

Comment:
Please stay idempotent and just overwrite with whatever is given

Reply:
Do you mean to always generate sops file?

Reply:
always overwrite with what is configured imo...ppl might want to switch

Reply:
I am sorry but I do not understand you